2-copy-of-code/lesson-14/scripts/checkout.js

```diff
@@ -109,4 +109,13 @@ document.querySelectorAll('.js-delete-link')
       );
       container.remove();
     });
-  });
\ No newline at end of file
+  });
+
+let cartQuantity = 0;
+
+cart.forEach((cartItem) => {
+  cartQuantity += cartItem.quantity;
+});
+
+document.querySelector('.js-return-to-home-link')
+  .innerHTML = `${cartQuantity} items`;
```

Comment:
I am getting a Nan Value from cartQuantity

Reply:
function calculateCartQuantity() {
  let cartTotal = 0;
  cart.forEach(cartItem => {
    cartTotal += cartItem.quantity;
  })
  const returnHomeLink = document.querySelector('.js-return-to-home-link');
  returnHomeLink
    .innerHTML = `${cartTotal} items`
  return cartTotal;
}
you can try this, if his solution doesn't work